target/scripts/start-mailserver.sh

```diff
@@ -82,8 +82,10 @@ function _register_functions() {
   _register_setup_function '_setup_opendmarc' # must come after `_setup_opendkim`
   _register_setup_function '_setup_policyd_spf'
 
+  _register_setup_function '_setup_logrotate'
+
   _register_setup_function '_setup_security_stack'
-  _register_setup_function '_setup_rspamd'
+  _register_setup_function '_setup_rspamd' # must come after _setup_logrotate
```

Comment:
I assume this is due to your addition of `export` reliance?

Reply:
Indeed, and it can be dropped now since we panic anyway.